apps/api/src/app/environments/render/email-schema-extender.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-param-reassign */
+import { TipTapNode } from '@novu/shared';
+
+// Define the command interface
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface ExpendEmailEditorSchemaCommand {
+  schema: TipTapNode;
+}
+
+// Rename the class to ExpendEmailEditorSchemaUseCase
+export class ExpendEmailEditorSchemaUseCase {
+  execute(command: ExpendEmailEditorSchemaCommand): TipTapNode {
+    return this.expendSchema(command.schema);
+  }
+
+  private expendSchema(schema: TipTapNode): TipTapNode {
+    const content = schema.content!.map(this.processNodeRecursive.bind(this)).filter(Boolean) as TipTapNode[];
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is there a case where `schema.content` could be falsy? Perhaps we should we throw an exception here instead to avoid a runtime `TypeError` if it's not defined.‏

Reply:
i'm pushing this comment to the next task, this functionlality is now broken and need a revision